tool/site/client/src/services/test-packs/index.ts

```diff
@@ -208,6 +236,12 @@ function createPrescription(
   return fhirPrescription
 }
 
+function pad(value: string, size: number) {
+  value = value.toString()
+  while (value.length < size) value = "0" + value
```

Comment:
just double checking 0s are at the front and not the end for ods codes? should this padding apply when you are talking about a 3-character ods code (parent org) and a 5/6-character ods code (child org)?

Reply:
it's from the prescription type scenarios we have to prove out. the short form prescription id needs to contain the org code in the middle of it. in primary care the ods code should be the correct length as that's what the hl7v3 api was built around, but in other care settings ods codes can be (and usually are) shorter, for these cases we need to 0 pad from the start of the odscode so that the short form prescription id is valid

Reply:
cool just wanted to make sure there was a clear reason